hw/ip/soc_dbg_ctrl/data/soc_dbg_ctrl.hjson

```diff
@@ -0,0 +1,483 @@
+                    exchange cannot take place anymore until next reset. Note that
+                    the rest of the SoC reset sequence is triggered by the OT RoT"
+                  '''
+            resval: "0"
+          }
+          { bits: "6"
+            name: "auth_unlock_success"
+            desc: ''' 
+                    Status bit indicating whether the unlock protocol 
+                    resulted in a successful unlock at requested level"
+                  '''
+            resval: "0"
+          }
+          { bits: "7"
+            name: "auth_unlock_failed"
+            desc: ''' 
+                    Status bit indicating whether the unlock protocol was
+                    ulted in unlock failure at requested level"
+                  '''
+            resval: "0"
+          }
+        ]
+      }
+      { name: "JTAG_BOOT_STATUS"
+        desc: '''
+              Debug boot status register that tells important boot state information.
+              Note that this information is reflected only if the hw_dft_en signal is true.
+              '''
+        swaccess: "ro"
+        hwaccess: "hwo"
+        hwext:    "true"
+        fields: [
+          { bits: "0"
+            name: "main_clk_status"
+            desc: ''' 
+                    Status of the main clock activation
+                  '''
+            resval: "0"
+          }
+          { bits: "1"
+            name: "io_clk_status"
+            desc: ''' 
+                    Status of the IO Clock activation
+                  '''
+            resval: "0"
+          }
+          { bits: "2"
+            name: "otp_done"
+            desc: ''' 
+                    OTP controller initialization complete
+                  '''
+            resval: "0"
+          }
+          { bits: "3"
+            name: "lc_done"
+            desc: ''' 
+                    Lifecycle controller initialization done; LC policy is decoded and set
+                  '''
+            resval: "0"
+          }
+          { bits: "4"
+            name: "cpu_fetch_en"
+            desc: ''' 
+                    Indication from powermanger to IBEX to state code execution
+                  '''
+            resval: "0"
+          }
+          { bits: "11:5"
+            name: "halt_fsm_state"
+            desc: '''
+                  The state of the halt state FSM.
+                  '''
+            resval: "0"
+          }
+          { bits: "14:12"
+            name: "rom_ctrl_done"
+            desc: ''' 
+                    Rom control initialization done; One bit corresponding to each ROM 
+                    ROM0 = base ROM 
+                    ROM1 = second ROM partition
+                    ROM2 = the continue boot inidcaiton form IBEX halt FSM socdbg_ctrl module - 
+                          leverages pwrmgr ROM controler inputs to halt boot sequence
+                  '''
+            resval: "0"
+          }
```

Comment:
The "ROM2" bit isn't really a ROM. I don't think this is a problem for the implementation: we should document it properly and then can expect DD/DV engineers to understand the mechanism. But for users/SW I think we should expose a cleaner interface. So how about something along the lines of
```suggestion
          { bits: "14:12"
            name: "boot_greenlight_done"
            desc: ''' 
                    Green lights for the boot process: _done_ indication coming from
                    [0]: base ROM
                    [1]: second ROM
                    [2]: this module
                    
                    Note that for the boot process to go through, all _done_ bits in this field and all _good_ bits in the next field need to be set.
                  '''
            resval: "0"
          }
```
And then similarly change `rom_ctrl_good` to `boot_greenlight_good` below